src/fides/api/alembic/migrations/versions/c9c72b3d550b_data_migration_converting_existing_.py

```diff
@@ -0,0 +1,214 @@
+"""data migration converting existing 'Cookies' resources to 'Asset' resources with type 'Cookie'
+
+Revision ID: c9c72b3d550b
+Revises: c961528edfc6
+Create Date: 2025-02-18 18:33:56.039924
+
+"""
+
+import uuid
+from typing import Dict
+
+import psycopg2
+import sqlalchemy as sa
+from alembic import op
+from loguru import logger
+
+from fides.api.models.asset import Asset
+
+# revision identifiers, used by Alembic.
+revision = "c9c72b3d550b"
+down_revision = "c961528edfc6"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    try:
+        op.add_column("asset", sa.Column("description", sa.String(), nullable=True))
+    except psycopg2.errors.DuplicateColumn:
+        logger.warning(
+            "Column 'description' already exists in 'asset' table. Skipping column addition."
+        )
```

Comment:
hm, seems weird we'd need to handle this? i suppose there's no harm in being extra safe

Reply:
Mostly a "just in case". I ran into it once or twice migrating down/up/down/up and I figured it wouldn't hurt